src/osp/Active/SysNewton.h

```diff
@@ -285,9 +263,8 @@ class SysNewton : public IDynamicSystem
 };
 
 template<class TRIANGLE_IT_T>
-void SysNewton::shape_create_tri_mesh_static(ACompCollisionShape &shape,
-                                             TRIANGLE_IT_T const& start,
-                                             TRIANGLE_IT_T const& end)
+void SysNewton::shape_create_tri_mesh_static(ACompShape& rShape,
+    ACompCollider& rCollider, TRIANGLE_IT_T const& start, TRIANGLE_IT_T const& end)
```

Comment:
just take `start` by value, iterate it directly, and skip creating `TRIANGLE_IT_T it`

Reply:
I didn't write this function, I just updated it since I changed how the colliders work; I'd prefer not to screw with it here.